content-src/components/ManualMigration/ManualMigration.jsx

```diff
@@ -1,49 +0,0 @@
-import {actionCreators as ac, actionTypes as at} from "common/Actions.jsm";
-import {connect} from "react-redux";
-import {FormattedMessage} from "react-intl";
-import React from "react";
-
-/**
- * Manual migration component used to start the profile import wizard.
- * Message is presented temporarily and will go away if:
- * 1.  User clicks "No Thanks"
- * 2.  User completed the data import
- * 3.  After 3 active days
- * 4.  User clicks "Cancel" on the import wizard (currently not implemented).
- */
-export class _ManualMigration extends React.PureComponent {
-  constructor(props) {
-    super(props);
-    this.onLaunchTour = this.onLaunchTour.bind(this);
-    this.onCancelTour = this.onCancelTour.bind(this);
-  }
-
-  onLaunchTour() {
-    this.props.dispatch(ac.AlsoToMain({type: at.MIGRATION_START}));
-    this.props.dispatch(ac.UserEvent({event: at.MIGRATION_START}));
-  }
-
-  onCancelTour() {
-    this.props.dispatch(ac.AlsoToMain({type: at.MIGRATION_CANCEL}));
-    this.props.dispatch(ac.UserEvent({event: at.MIGRATION_CANCEL}));
-  }
-
-  render() {
-    return (<div className="manual-migration-container">
-        <p>
-          <span className="icon icon-import" />
-          <FormattedMessage id="manual_migration_explanation2" />
```

Comment:
The icon and strings will be unused, so we can probably remove those?

Reply:
Oh the strings. Do we need to wait a couple of releases before removing them?

Reply:
Nah, strings still exist on beta, and localizers see a unified view of all strings across channels.

Reply:
ooh, it was system addon days where we had to wait I guess. good old days

Reply:
and by system addon i guess i meant test pilot 😉